sound/soc/sof/sof-spi-dev.c

```diff
@@ -96,11 +77,12 @@ static int sof_spi_probe(struct spi_device *spi)
 	dev_dbg(dev, "created machine %s\n",
 		dev_name(&sof_pdata->pdev_mach->dev));
 
-	/* continue probing after firmware is loaded */
-	ret = request_firmware_nowait(THIS_MODULE, true, mach->sof_fw_filename,
-				      dev, GFP_KERNEL, priv, sof_spi_fw_cb);
-	if (ret)
+	/* register sof-audio platform driver */
+	ret = sof_create_platform_device(priv);
+	if (ret) {
 		platform_device_unregister(sof_pdata->pdev_mach);
+		dev_err(dev, "error: failed to create platform device!\n");
+	}
```

Comment:
but not here, the runtime_pm is not skipped.

Reply:
should be skipped, let me correct it.